package.json

```diff
@@ -282,5 +283,142 @@
       "atom",
       "snapshotResult"
     ]
+  },
+  "versionTracker": {
+    "track": [
+      {
+        "name": "node",
+        "versionCommands": [
+          [
+            "node",
+            "--version"
+          ]
+        ]
+      },
+      {
+        "name": "npm",
+        "versionCommands": [
+          [
+            "npm",
+            "-v"
+          ]
+        ]
+      },
+      {
+        "name": "xcode",
+        "versionCommands": [
+          [
+            "/usr/bin/xcodebuild",
+            "-version"
+          ]
+        ]
+      },
+      {
+        "name": "clang",
+        "versionCommands": [
+          [
+            "clang",
+            "--version"
+          ]
+        ]
+      },
+      {
+        "name": "visual studio",
+        "versionCommands": [
+          [
+            "powershell",
+            ".\\script\\get-visual-studio-verion.ps1"
+          ],
+          [
+            "powershell",
+            "./script/get-visual-studio-verion.ps1"
+          ],
+          [
+            "pwsh",
+            ".\\script\\get-visual-studio-verion.ps1"
+          ],
```

Comment:
Why do have two versions of this? The forward slash one is enough for all the operating systems.

Reply:
If its tested and works without the extra's that would be great. The code is still using spawn which doesn't invoke a shell, but I guess since its calling powershell powershell will convert the argument